packages/editor-ui/src/views/NodeView.vue

```diff
@@ -3060,6 +3060,8 @@ export default defineComponent({
 
 				const promptResponse = (await promptResponsePromise) as MessageBoxInputData;
 
+				if (promptResponse?.action !== 'confirm') return;
```

Comment:
I just don't like the magic string here but as far as I remember there isn't an enum or something we can use from element-ui to check the `action`. I think when we use prompt we usually check the `value`. Or is it not applicable here (checking if `value` has length)?

However we have some constants for the confirm actions [here](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/constants.ts#L308) (not happy with the naming though)

Reply:
Good point, switched to the `MODAL_CONFIRM` enum

Reply:
Thanks, until we have better this will do it